contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2153,6 +2153,29 @@ class tsqlMutator : public TSqlParserBaseListener
 	    stream.setText(name->start->getStartIndex(), str.c_str());
 	}
     }
+
+	void exitFunction_call(TSqlParser::Function_callContext *ctx) override
+	{
+		if (ctx->func_proc_name_server_database_schema())
+		{
+			auto fpnsds = ctx->func_proc_name_server_database_schema();
+
+			if (fpnsds->DOT().empty() && fpnsds->id().back()->keyword()) /* built-in functions */
+			{
+				auto id = fpnsds->id().back();
+
+				if (id->keyword()->SUBSTRING()) /* NULLIF */
```

Comment:
1. Please fix the comment
2. We need to fix NULLIF (and probably for CHECKSUM) as well. Please create a JIRA for that.